### PR TITLE
Add Services and DTOs, AutoMapper

### PR DESCRIPTION
**Pull request by temshik**
Added Interfaces of Services and their implementation, Dtos of entities

### CODE REVIEW

File: src/Application/DTOs/AppUserDTO/UpdateAppUserDTO.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        public string TelephoneNumber { get; set; }
+
+        [Required(ErrorMessage = "This field {0} is required")]
+        [StringLength(150, ErrorMessage = "The field {0} must be between {2} and {1} characters", MinimumLength = 2)]
+        public string Email { get; set; }
+        public Roles Role { get; set; }
+        public DateTime EmploymentStart { get; set; }
+        public DateTime? EmploymentEnd { get; set; }
+        public Guid? PrefferdWorkPlaceId { get; set; }
+        public bool isDeleted { get; set; }
```
Comment (svadrif) [inline src/Application/DTOs/AppUserDTO/UpdateAppUserDTO.cs:28]: change isDeleted to IsDeleted
Or you can use IsActive

File: src/Application/Profiles/AppUserProfile.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+using System.Threading.Tasks;
+
+namespace Application.Profiles
+{
+    public class AppUserProfile : Profile
+    {
+        public AppUserProfile()
+        {
+            CreateMap<AppUser, GetAppUserDTO>()
+                .ForMember(dest => dest.Id, src => src.MapFrom(x => x.Id))
```
Comment (svadrif) [inline src/Application/Profiles/AppUserProfile.cs:17]: If I'm not mistaken, automapper automatically maps entity's properties to dto properties if they have the same type and name

Comment (DoniyorJN) [reply src/Application/Profiles/AppUserProfile.cs:17]: I requested to map explicit mapping

File: src/Application/DTOs/WorkPlaceDTO/GetWorkPlaceDTO.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using Domain.Enums;
+
+namespace Application.DTOs.WorkPlaceDTO
+{
+    public class GetWorkPlaceDTO
+    {
+        public Guid Id { get; set; }
+        public string Number { get; set; }
+        public SeatsType Type { get; set; }
+        public bool NextToWindow { get; set; }
```
Comment (svadrif) [inline src/Application/DTOs/WorkPlaceDTO/GetWorkPlaceDTO.cs:10]: Rename NextToWindow to IsNextToWindow

**Review by svadrif — APPROVED** [review]

**Pull request merged**